bin/concept-of-the-week.txt

```diff
@@ -1 +1 @@
-content/html/concepts/classes/classes.md
\ No newline at end of file
+content/general/concepts/what-is-a-parameter/what-is-a-parameter.md
```

Comment:
Is this file meant to be in this PR?

Reply:
> Is this file meant to be in this PR?

Hi @lukeecart , no it's not supposed to be there, I have no idea how that came along.

Reply:
No worries, @lukeecart! @yangc95 found a fix! 👍